### PR TITLE
Add styles for the "post comment" page to the product

### PR DESCRIPTION
**Pull request by AlejandroCuriel**
# Description
 - What?: Some styles are added to the post product comment page
 - Why?: To give it styles and not show the form in a way that the user feels comfortable. See #67  for more information.
 
## Screenshots 
Screenshot from 2022-05-24 20-49-25

### CODE REVIEW

**Review by nallely64 — DISMISSED** [review]
Lgtm!

File: app/assets/stylesheets/base/_globals.scss [hunk 1]
```diff
@@ -14,10 +14,13 @@ body {
...
 
+textarea {
+  resize: none;
+}
+
 .main-container {
   min-height: calc(100vh - $footer-size - 56px);
   width: 100%;
   max-width: 1480px;
   margin: 0 auto;
```
Comment (JuanequeX) [inline app/assets/stylesheets/base/_globals.scss:25]: Sort alphabetically this block, please!

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  margin: 2rem;
+  max-width: 100px;
+  padding: 0.5rem 0.5rem;
+  text-align: center;
+
+  a {
+    color: white;
+    text-decoration: none;
+  }
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:59]: Add a blank line here please

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+.form__button--back {
+  background-color: $red;
+  margin-left: 2rem;
+  margin: 2rem;
+  max-width: 100px;
+  padding: 0.5rem 0.5rem;
+  text-align: center;
+
+  a {
+    color: white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:56]: Use a variable to add this color, please

File: app/assets/stylesheets/pages/_post_comment.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+.form-actions-comment {
+  .btn {
+    background-color: $links;
+    font-weight: 700;
+    margin-bottom: 1.3rem;
+  }
+
+  .btn:hover {
+    background-color: #caa006;
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_post_comment.scss:42]: Use a variable to add this color, please

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/post_comments/_form.html.erb [hunk 5]
```diff
@@ -3,12 +3,13 @@
   <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
 
   <div class="form-inputs">
-    <%= f.input :comment %>
+    <%= f.input :comment, label: "Comentario: ", :input_html => { :rows => 3 , :style => 'width: 100%'} %>
```
Comment (YilKanda) [inline app/views/post_comments/_form.html.erb:6]:
```suggestion
    <%= f.input :comment, label: "Comentario: ", input_html: { rows: 3 , style: 'width: 100%'} %>
```

**Review by JuanequeX — APPROVED** [review]
LGTM

**Comment by JuanequeX** [comment]
Solve your merge conflict using `rebase --continue`

File: app/views/post_comments/_form.html.erb [hunk 6]
```diff
@@ -3,12 +3,21 @@
...
   </div>
 
-  <div class="form-actions">
-    <%= f.button :submit %>
+  <div class="product__information">
+    <h1>Titulo del producto</h1>
+    <p>Imagen del producto</p>
   </div>
+
+
```
Comment (YilKanda) [inline app/views/post_comments/_form.html.erb:22]: Remove the extra space.

**Review by YilKanda — DISMISSED** [review]
LGTM!

**Review by EliasUR — APPROVED** [review]
LGTM

**Review by Javi1499 — APPROVED** [review]

**Review by EliasUR — DISMISSED** [review]
LGTM

**Review by germanruzca — APPROVED** [review]

**Pull request merged**